light-zk.js/tests/transactionParameters.test.ts

```diff
@@ -24,13 +24,14 @@ import {
   AUTHORITY,
   Utxo,
   Account,
-  TRANSACTION_MERKLE_TREE_KEY,
   IDL_VERIFIER_PROGRAM_ZERO,
   IDL_VERIFIER_PROGRAM_ONE,
   IDL_VERIFIER_PROGRAM_TWO,
+  MerkleTreeConfig,
 } from "../src";
 import { TOKEN_PROGRAM_ID } from "@solana/spl-token";
 import { bs58 } from "@coral-xyz/anchor/dist/cjs/utils/bytes";
+import { mock } from "node:test";
```

Comment:
is this import on purpose?

Reply:
No, sorry. :( We really need to find some linter which complains about unused imports in TS.